src/basic/generate-af-list.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/sh
 
-cpp -dM -include sys/socket.h - </dev/null | \
+$1 -E -dM -include sys/socket.h - </dev/null | \
```

Comment:
You should definitely quote this as `"$1"` else things will break if the path to the compiler has spaces in it.

However, you should also consider using `"$@"` as I suggested because the compiler is an *array*, because for instance someone might be using distcc and/or ccache or some custom wrapper.